ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -694,6 +694,64 @@ literals:
   - PRESENT
   - ABSENT
 
+# Added for TPKD day2
+- condensing_supply_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- condensing_return_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- chilled_supply_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- chilled_return_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- oil_heater_status:
+  - ON
+  - OFF
+- inhibit_start_time_sensor
+- load_limit_capacity
```

Comment:
Load being weight? Does this mean this is the weight capacity of this machine? (See your definition below.)

Reply:
The definition of load has been changed

Reply:
I still don't get what this is. I assume this is chiller cooling load? We have `cooling_thermal_power_capacity` already for cooling capacity (e.g. the chiller is rated for 100 tons). What does this field represent?

Reply:
changed to limit_percentage_capacity.

Reply:
it is the demand for the chiller

Reply:
So can this be replaced with `cooling_thermal_power_capacity`? That is what is used for chiller capacity (tons or KW); demand would be a `cooling_thermal_power_sensor`.

Reply:
The point can be replaced with cooling_thermal_power_capacity.